### PR TITLE
Multimodal Launcher Merge

### PR DESCRIPTION
**Pull request by yaoyu-33**
PR for adding multimodal models

### CODE REVIEW

**Comment by dimapihtar** [comment]
@yaoyu-33 could you resolve all the conflicts, please?

**Comment by yaoyu-33** [comment]
> @yaoyu-33 could you resolve all the conflicts, please?

Done!

**Review by dimapihtar — APPROVED** [review]
LGTM. Thank you!

File: launcher_scripts/data/nsfw/concepts.txt
Comment (maanug-nv) [inline launcher_scripts/data/nsfw/concepts.txt:1]: Is this file meant to be here? Normally files under 'data' are downloaded by user from some source. Just wondering.

**Pull request merged**